pages/search/search.scss

```diff
@@ -16,4 +16,22 @@
     display: flex;
     align-items: center;
   }
+
+  .search-bar input {
```

Comment:
Is this supposed to _only_ target the search bar, or should it be an update for all form fields?

Reply:
Just checked the designs, this should be for all form fields.I'll update this.